packages/nodejs/src/__tests__/utils.test.ts

```diff
@@ -9,4 +10,26 @@ describe("Utils", () => {
       expect(timestamp.nsec).toEqual(108000000)
     })
   })
+
+  describe("hrTime", () => {
+    it("returns a second/nanosecond tuple", () => {
+      const performance = {
+        timeOrigin: 1648719688126.149,
+        now: function () {
+          return 116931.83823999763
+        },
+        clearMarks: perf_hooks.performance.clearMarks,
+        mark: perf_hooks.performance.mark,
+        measure: perf_hooks.performance.measure,
+        nodeTiming: perf_hooks.performance.nodeTiming,
+        timerify: perf_hooks.performance.timerify,
+        eventLoopUtilization: perf_hooks.performance.eventLoopUtilization
```

Comment:
```suggestion
        ...perf_hooks.performance,
        timeOrigin: 1648719688126.149,
        now: function () {
          return 116931.83823999763
        }
```

Easier to only customize what you really want to change without the noise of all the things you want to keep the default. And if any new things are added in the future we don't need to manually update it there.